lib/core/src/Cardano/Wallet.hs

```diff
@@ -2007,10 +2174,10 @@ selectAssets ctx pp params transform = do
     let selectionParams = SelectionParams
             { assetsToMint =
                 params ^. (#txContext . #txAssetsToMint)
-            , assetsToBurn =
-                params ^. (#txContext . #txAssetsToBurn)
             , extraCoinIn = Coin 0
             , extraCoinOut = Coin 0
+            , assetsToBurn =
+                params ^. (#txContext . #txAssetsToBurn)
```

Comment:
Any reason why this one gets moved, but not `assetsToMint`? 😄

Reply:
Oops. Might have happened during a rebase. Fixed.